students/YingGuo/session04/file_lab.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+def function1():
+    """a program which prints the full path for all files in the current directory, one per line"""
+    import pathlib
```

Comment:
Minor: All import statements are expected to be at the top of any script (after line 1).